### DISCUSSION TITLE
Using safetensors model

### DISCUSSION THREAD

**Discussion by MiggerX2** [imartinez/privateGPT#1753] [General]
Hello, is it possible to use [this](https://huggingface.co/sdadas/polish-gpt2-xl/tree/main) model with privateGPT and work with embeddings (PDFS,etc.) at the same time?
Or privategpt doesn't accept safetensors and only works with .gguf?

Thanks in advance, I'm absolute noob and I want to just be able to work with documents in my local language (Polish)

**Comment by CrazybutSolid** [comment]
same q from me!

**Comment by haexhub** [comment]
any news on this?
